daktari/checks/direnv.py

```diff
@@ -26,9 +27,22 @@ def check(self) -> CheckResult:
         )
 
 
+class EnvrcContainsText(Check):
+    name = "direnv.envrc.containsText"
+
+    def __init__(self, expected_string: str, suggestion: str):
+        self.file_path = f"{getcwd()}/.envrc"
+        self.expected_string = expected_string
+        self.pass_fail_message = f"{self.file_path} does <not/> contain '{expected_string}'"
+        self.suggestions = {OS.GENERIC: suggestion}
+
+    def check(self) -> CheckResult:
+        return self.verify(file_contains_text(self.file_path, self.expected_string), self.pass_fail_message)
+
+
 class DirenvAllowed(Check):
     name = "direnv.allowed"
-    depends_on = [DirenvInstalled]
+    depends_on = [DirenvInstalled, EnvrcContainsText]
```

Comment:
Where are we passing in the string to check for?

Reply:
In the constructor. It's `expected_string` in the `__init__` method.

Reply:
Sorry - ignore me. I thought this PR was somewhere else. Looks good!